pilot/pkg/model/authorization.go

```diff
@@ -30,162 +29,168 @@ type RolesAndBindings struct {
 	// ServiceRoles in the same namespace.
 	Roles []Config
 
-	// Maps from ServiceRole name to its associated ServiceRoleBindings.
-	RoleNameToBindings map[string][]*rbacproto.ServiceRoleBinding
+	// ServiceRoleBindings indexed by its associated ServiceRole's name.
+	Bindings map[string][]*rbacproto.ServiceRoleBinding
 }
 
-// AuthorizationPolicyConfig stores the AuthorizationPolicy and its name.
-type AuthorizationPolicyConfig struct {
-	Name string
+// AuthorizationPolicies organizes authorization policies by namespace.
+type AuthorizationPolicies struct {
+	// Maps from namespace to RolesAndBindings.
+	namespaceToV1Policies map[string]*RolesAndBindings
+
+	// The mesh global RbacConfig.
+	rbacConfig *rbacproto.RbacConfig
 }
 
-// AuthorizationConfigV2 stores a list of AuthorizationPolicyConfig and ServiceRole in a given namespace.
-type AuthorizationConfigV2 struct {
-	// A list of AuthorizationPolicyConfig.
-	AuthzPolicies []*AuthorizationPolicyConfig
+// NewAuthorizationPolicies creates a new AuthorizationPolicies.
+func NewAuthorizationPolicies(env *Environment) (*AuthorizationPolicies, error) {
```

Comment:
'New' is a bit misleading. Maybe replace with 'Collect' ?

Reply:
I feel 'Get' might be simpler?